src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -206,499 +175,437 @@ protected int loadItems(Connection conn, int itemStartInclusive, int itemEndIncl
                 itemPrepStmt.addBatch();
                 batchSize++;
 
-                if (batchSize == TPCCConfig.configCommitCount) {
+                if (batchSize == workConf.getBatchSize()) {
                     itemPrepStmt.executeBatch();
                     itemPrepStmt.clearBatch();
-                    transCommit(conn);
                     batchSize = 0;
                 }
-            } // end for
-
-
-            if (batchSize > 0) itemPrepStmt.executeBatch();
-            transCommit(conn);
-
-        } catch (BatchUpdateException ex) {
```

Comment:
The exception handling as rollback, commit wrapper leading to rollback, were removed

Reply:
ok